docs/wire-tool-v1.0/user-manual/index.md

```diff
@@ -452,22 +452,22 @@ Mounting the WT-01 in its base configuration on the Ender 3 is a simple task. Th
 
 ## Threading of Wire
 
-Threading is the act of guiding the brass EDM wire through the wire EDM tool. Here are a few tip for threading the wire tool: 
+Threading is the act of guiding the brass EDM wire through the wire EDM tool. Here are a few tips for threading the wire tool: 
 
-- Use a lighter to cut the brass EDM wire, not a clipper or knife. Clipping or cutting the wire with a tool leaves a bur on the wire, which prevents it from entering the ceramic wire guide. Burs can be avoided by placing the EDM wire into the flame of a lighter, and applying tension by hand. The heat melts the wire, and allows it to be drawn into a sharp point. This sharp point permits the brass EDM wire to enter the precision wire guide. 
-- Release tension on the system by loosening tension spring completely. Depress tension arm and manually feed wire through PTFE tube and wire guide.
-- Guide wire around first and second idler taking care to ensure wire sits below WC block.
-- Thread wire through hole in waste spool. Pull tight and hold wire out to the side near the axis of rotation of the waste spool.
+- Use a lighter to part the brass EDM wire, DO NOT cut it with a clipper or knife. Clipping or cutting of the wire leaves a burr on the wire, which prevents it from entering the ceramic wire guide. Burrs can be avoided by placing the EDM wire into the flame of a lighter, and applying tension by hand. The heat melts the wire, and allows it to be drawn into a sharp point. This sharp point permits the brass EDM wire to enter the precision wire guide. 
```

Comment:
I think "cut" was a better descriptor than "part"

Reply:
If you guys all agree with that that's fine and you can leave it as cut. I changed it from cut because you're not actually cutting it more stretching it till it breaks so I thought that was more descriptive but either way is fine with me.

Reply:
This is definitely clearer. Appreciate the help!